zqd/handlers_test.go

```diff
@@ -0,0 +1,121 @@
+package zqd_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zio/bzngio"
+	"github.com/brimsec/zq/zio/zngio"
+	"github.com/brimsec/zq/zng/resolver"
+	"github.com/brimsec/zq/zqd"
+	"github.com/brimsec/zq/zqd/api"
+	"github.com/brimsec/zq/zqd/space"
+	"github.com/brimsec/zq/zql"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSimpleSearch(t *testing.T) {
+	src := `#0:record[_path:string,ts:time,uid:bstring]
```

Comment:
Would you please start the characters for this raw string on the next line? It's a little easier to read.

Reply:
sure!